packages/testsuite/cypress/e2e/mail/test-configuration-subsystem-mail-server.cy.ts

```diff
@@ -0,0 +1,206 @@
+describe("TESTS: Configuration => Subsystems => Mail", () => {
```

Comment:
Please append `=> Server => Attributes` since we're testing the Attributes tab on the Server navigation item